conf/ocsci/tainted_worker_nodes_ui.yaml

```diff
@@ -0,0 +1,3 @@
+---
+DEPLOYMENT:
+    ocs_operator_nodes_to_taint: 3
```

Comment:
I'm still wondering if we can use the existing taints yamls instead of creating a new one 
conf/ocsci/tainted_worker_nodes.yaml

@OdedViner I think you combined two different changes in the same PR
1) Enable LSO UI deployment on ODF 4.10
2) Enable taints during UI deployment

I would suggest you to open a separate PR for taints as I feel it would need some discussions to proceed further.

Reply:
I agree with @PrasadDesala here. Please separate to two different PRs as those are two different problems/issues which should be handled separately.

Reply:
Also when we discussed in the mail. If we will have tainted nodes, we should probably aways have extra worker nodes. Currently we are testing tainted nodes mainly with infra nodes. Which is in this config:  https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocsci/tainted_worker_nodes.yaml and production config files we have for testing using the same structure. E.g. https://github.com/red-hat-storage/ocs-ci/blob/master/conf/deployment/vsphere/upi_1az_rhcos_vmfs_3m_3w_3i.yaml .  In the case it will be UI deployment, you should just enable UI deployment and reuse current config I think, only when enabled via UI you will handle the deployment via UI and then you need to make sure you are selecting only those infra nodes.  @keesturam @ebenahar @hnallurv   WDYT?